hardhat.config.ts

```diff
@@ -48,7 +50,7 @@ const config: HardhatUserConfig = {
     strict: false,
   }*/
   mocha: {
-    timeout: 20000,
+    timeout: 50000,
```

Comment:
consider https://mochajs.org/#test-level

Reply:
👆🏽

Reply:
reverted this to how it was.

Reply:
my suggestion was rather to remove it from the config altogether.

Reply:
I removed that configuration at all, because it isn't necessary at the moment.